docs/software/camera.md

```diff
@@ -105,6 +114,15 @@ from any RPi camera, USB camera or an IP camera network stream.
     sudo pip2 install -U motioneye
     ```
 
+=== "Troubleshooting"
+
+    The camera should be activated by the installation process. This can be checked via `dietpi-config`: Select `Display Options`, then check for `RPi Camera` set to `[On]`. This should lead to a line `start_x=1` (without a leading `#`) in the file `/boot/config.txt`.  
+    Additionally the behaviour of the cameras LED can be set in the same dialog via `RPi Camera LED`.
```

Comment:
Better this?

```
=== "RPi camera module"

    If you have an official Raspberry Pi camera module, it should be activated by the installation process. This can be checked/enabled via `dietpi-config` > `Display Options` > `RPi Camera` to show `[On]`.  
    Additionally the behaviour of the cameras LED can be set in the same dialog via `RPi Camera LED`.

    Remark: After changing the camera activation you need to reboot and/or sometimes power cycle the SBC incl. camera.
```

Reply:
With those two installs it is not enabled automatically, would be also pretty wrong as we cannot know whether the user actually has and wants to use it with the RPi camera module or with a USB or other type of camera 😉.

Reply:
Ok, ok. I understand that I have to change the sentence for MotionEye and mjpeg-streamer.